jenkins/databricks/init_cudf_udf.sh

```diff
@@ -61,9 +47,41 @@ REQUIRED_PACKAGES=(
   requests
   sre_yield
 )
+if base=$(conda info --base); then
```

Comment:
Support install cudf-py packages via either Conda or PIP.

Reply:
Q: if no conda in PATH this would error out?

Reply:
Does any of Databricks Runtime 13.3+ have conda? If no, I think we can drop supporting conda installation.

Reply:
> Q: if no conda in PATH this would error out? 
> 

The if expression will not error out but get 'false' value instead.


> can we try to use the the `command` to check if exist first? this would help make the if expression and else branches more clear
> 
> ```
> if command -v conda >/dev/null 2>&1; then
> ```

Sounds good to me, let me update it to make the if/else more readable

Reply:
> Does any of Databricks Runtime 13.3+ have conda? 

DB-13.3+ do not install conda by default

> If no, I think we can drop supporting conda installation.

I'd like to keep conda installation there for some time, because:
Keep it to maintain compatibility with DB13.3 or earlier conda installations.
We also have the jenkins/databricks/cudf_udf_test.sh script, which is still compatible with conda installations.